src/main/java/mate/controller/GetAllDriversController.java

```diff
@@ -0,0 +1,25 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/drivers/all")
+public class GetAllDriversController extends HttpServlet {
+    private final DriverService driverService = (DriverService) AddDriverController.injector
+            .getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        List<Driver> allDrivers = driverService.getAll();
+        req.setAttribute("drivers", allDrivers);
+        req.getRequestDispatcher("/WEB-INF/views/all.jsp").forward(req, resp);
```

Comment:
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/drivers/all.jsp").forward(req, resp);
```

Лучше создать отдельные папки для jsp страниц, которые относятся к машинам, производителям или водителям. Так будет легче но ним ориентироваться в будующем